### PR TITLE
Assignment#1 (Prime Numbers)

### PR DESCRIPTION
**Pull request by sairaAnwaar29**
Task:
Write a program in PHP that takes two integers and returns all the prime numbers between those two (including them).

### CODE REVIEW

**Review by hamza-hafeez1 — APPROVED** [review]
LGTM

File: prime_numbers.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+$number1 = (int) readline("Enter first number \n");
+$number2 = (int) readline("Enter second number \n");
+
+$prime_numbers = prime_numbers($number1, $number2);
+
+echo "All the prime numbers are: ";
+
+foreach ($prime_numbers as $number) {
+    echo "{$number}" . "\n";
```
Comment (hamza-hafeez1) [inline prime_numbers.php:35]: {} added intentionally here?

Comment (sairaAnwaar29) [reply prime_numbers.php:35]: Yes. 
Just read somewhere that php doesn't know where the variable name is being ended. So, there are cases, when we need to explicitly mention where we need to end the variable name. For example, in the following case,

```
$degrees = 30.5;
echo $status = "The weather is $weather and temperature is {$degrees}º.";
```

So just to keep a practice, I have used curly braces here.

Comment (sairaAnwaar29) [reply prime_numbers.php:35]: Ok got it 👍

**Pull request merged**